src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/ttd/events/ui/FormLayout.json

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
unsure if we actually need the files under ui/ 
If we don't have any tests targeting the resourceController we might as well reduce the number of files included in the test projects

Reply:
The tests work without the ui files. SHould I delete all of them for all test apps?